### DISCUSSION TITLE
Generate Patch options (to remove line number)

### DISCUSSION THREAD

**Discussion by ourmaizcay** [WinMerge/winmerge#1183] [Q&A]
Hi - I'm relatively new user to WinMerge and have been looking at the Generate Patch Report options. 

I would like to know if it's possible to Generate Patch file - with just the different records without the reference to the line number. I thought I had done this previously but can't work out what settings I have used.

For example if 
File 1 has
line 1
line 2
line 3 
and 
File 2 
line 1
line 2
line 3
line 4

![FILES](https://user-images.githubusercontent.com/99322642/153161897-e6b3d023-6bd8-49dd-84c9-ce98845c56c2.PNG)
 
What if any settings can I use when I select  Tools - Generate Patch to generate file with just:
> line 4

No reference to 3a4 - as I currently see:
![OUTPUT](https://user-images.githubusercontent.com/99322642/153162579-8639c4b6-31c8-49cc-9526-0dea6bd05c75.PNG)

Appreciate any thoughts. Thanks

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have such option to get the output you expect.

If you want to get only the lines with differences, set DiffContext to 0, press Ctrl+A to select all, and copy them, as shown below.
![image](https://user-images.githubusercontent.com/98126/153193651-c5617aa7-84b6-4255-a3ba-36621e78c4d7.png)

**Comment by ourmaizcay** [comment]
Hi - thanks for your response. I must have been mistaken in thinking I had previously generated the output how I described. 

The Diff Context option achieves the same result - that's great. I was also using an old portable version that didn't have Diff Context.

Thanks for your help :-)
